libvirt/tests/src/mem/memory_allocation/memory_invalid_value.py

```diff
@@ -0,0 +1,88 @@
+
+from virttest import virsh
+
+from virttest.libvirt_xml import vm_xml
+from virttest.utils_test import libvirt
+from virttest.utils_libvirt import libvirt_vmxml
+
+
+def run(test, params, env):
+    """
+    Verify libvirt forbids invalid memory value
+
+    1.value: Zero/Minus
+    2.mem type: Memory/currentMemory/maxMemory/numa
+                mem currentMem numa mixed
+    """
+    def setup_test():
+        pass
+
+    def run_test():
+        """
+        Define vm and check msg
+        Start vm and check config
+        """
+        test.log.info("TEST_STEP1: Define vm and check result")
+        vmxml = vm_xml.VMXML.new_from_dumpxml(vm_name)
+        vmxml.setup_attrs(**vm_attrs)
+        test.log.debug("Define vm with %s." % vmxml)
+        cmd_result = virsh.define(vmxml.xml, debug=True)
+        if num == "minus":
+            libvirt.check_result(cmd_result, minus_error_msg)
+        elif num == "zero":
+            if error_msg:
+                libvirt.check_result(cmd_result, error_msg)
+            else:
+                test.log.info("TEST_STEP2: Check xml after define")
+                check_mem_config()
+                test.log.info("TEST_STEP3: Check start vm get correct result")
+                res = virsh.start(vm_name)
+                libvirt.check_exit_status(res, start_vm_error)
+                if start_vm_error:
+                    libvirt.check_result(res, start_error_msg)
```

Comment:
Should it be something like below?
```suggestion
                if start_error_msg:
                    libvirt.check_result(res, start_error_msg)
```